std_msgs/Float64MultiArray joint_config
---
string status
